### PR TITLE
Remove pug files

### PR DESCRIPTION
**Pull request by yenienserrano**
### Description
unused files are removed, after removing the pug-loader dependency
 
Issues Resolved
- #5059

### Test

Test the application and make sure it works as expected

### Check List
- [ ] All tests pass
  - [ ] `yarn test:jest`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -41,8 +41,9 @@ All notable changes to the Wazuh app project will be documented in this file.
 - Updated the `pdfmake` dependency to `0.2.6` [#4985](https://github.com/wazuh/wazuh-kibana-app/pull/4985)
 - The button to export the app logs is now disabled when there are no results, instead of showing an error toast [#4992](https://github.com/wazuh/wazuh-kibana-app/pull/4992)
 - Unify the SCA check result label name [#5031](https://github.com/wazuh/wazuh-kibana-app/pull/5031)
-- Updated `pdfmake, mocha and json5` dependencies [#5062](https://github.com/wazuh/wazuh-kibana-app/pull/5062)
-
+- Updated `mocha` dependency to `10.1.0` [#5062](https://github.com/wazuh/wazuh-kibana-app/pull/5062)
+- Updated `pdfmake` dependency to `0.2.7` [#5062](https://github.com/wazuh/wazuh-kibana-app/pull/5062)
+- Remove `pug-loader` dependency [#5062](https://github.com/wazuh/wazuh-kibana-app/pull/5062) [#5089](https://github.com/wazuh/wazuh-kibana-app/pull/5089)
```
Comment (Desvelao) [inline CHANGELOG.md:46]: **nitpick**: this entry should be moved to `Removed` section

**Comment by Desvelao** [comment]
# Changes

Moved the deletion of `pug-loader` dependency to the `Removed` section in the changelog

File: CHANGELOG.md (hunk 1 above)
Comment (Desvelao) [reply CHANGELOG.md]: Solved

**Review by Desvelao — APPROVED** [review]
**review**:

:green_circle: code
:green_circle: test

**Review by asteriscos — APPROVED** [review]
CR: :heavy_check_mark: 
Test: :heavy_check_mark:

**Pull request merged**